server/controllers/users/changePassword.js

```diff
@@ -1,3 +1,23 @@
-exports.changePassword = (req, res) => {
-  res.send('hello from changePassword file');
+const { hash, compare } = require('bcrypt');
+const { users: { changePassword, getUserById } } = require('../../database/queries');
+
+exports.changePassword = (req, res, next) => {
+  const { userId, oldPassword, newPassword } = req.body;
+  getUserById(userId).then(({ rows }) => {
+    if (rows.length === 0) throw new Error();
```

Comment:
just FYI, this will result in returning 400, in a RESTful API it should return 404. However the API should be designed differently:
`/users/:id/changePassword`
But that's a bigger task so don't bother, I'll probably create an issue for that over the weekend..

Reply:
Should we use camelCase or hyphens in RESTful API? @11janci